### PR TITLE
New update Open API json file

### PR DESCRIPTION
**Pull request by Silence0ne**
Fix bugs

### CODE REVIEW

**Review by al-abd — APPROVED** [review]

File: open-api.json [hunk 1]
```diff
@@ -1241,6 +1290,16 @@
...
               "type": "string"
             }
+          },
+          {
+            "in": "query",
+            "name": "format",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": ["text", "Word"],
```
Comment (nyzd) [inline open-api.json]: Please change 'Word' to 'word'

File: open-api.json [hunk 2]
```diff
@@ -1349,7 +1409,58 @@
...
+            }
+          },
+          {
+            "in": "query",
+            "required": false,
+            "name": "format",
+            "description": "The format of the ayahs.",
+            "schema": {
+              "type": "string",
+              "enum": ["Text", "Word"]
```
Comment (nyzd) [inline open-api.json]: These also should be lowercase

**Review by nyzd — CHANGES_REQUESTED** [review]

File: open-api.json (hunk 1 above)
Comment (Silence0ne) [reply open-api.json]: Thank you, i changed to word

File: open-api.json (hunk 2 above)
Comment (Silence0ne) [reply open-api.json]: Thank you, fixed

**Review by nyzd — APPROVED** [review]

**Pull request merged**